### ISSUE TITLE
Flutter CLI commands lose interactive prompts when run with fvm

### ISSUE THREAD

**Issue by techouse** [leoafarias/fvm#500]
**Describe the bug**
All the interactive Flutter CLI commands which usually prompt a user to select a device or the like don't work.

**To Reproduce**

1. Have multiple devices tethered to your computer
2. Run `fvm flutter run`
3. Command fails with the error:
```
More than one device connected; please specify a device with the '-d <deviceId>' flag, or use '-d all' to act on all devices.
```

**Expected behavior**
Flutter prompting me which device to select interactively.

**Desktop (please complete the following information):**

- OS: not applicable
- FVM Version: 2.4.1

**Comment by leoafarias** [comment]
@techouse this doesn't seem to be a case of FVM losing interactivity. Did you check with the standalone Flutter SDK, it seems the error message point for you to specify which device you would like to run on as there is more than one.

**Comment by techouse** [comment]
When runnings `flutter` without `fvm` I get a prompt

```
➜  app git:(flutter-3.7) ✗ flutter run --flavor nonprod
Multiple devices found:
Pixel 6 (mobile)           • SOME_ANDROID_ID_BLABLA  • android-arm64 • Android 13 (API 33)
techouse’s iPhone (mobile) • SOME_IPHONE_ID_BLABLA   • ios           • iOS 16.3 20D47
[1]: Pixel 6 (SOME_ANDROID_ID_BLABLA)
[2]: techouse’s iPhone (SOME_IPHONE_ID_BLABLA)
Please choose one (To quit, press "q/Q"): q
```

when running `fvm flutter` it just quits

```
➜  app git:(flutter-3.7) ✗ fvm flutter run --flavor nonprod
More than one device connected; please specify a device with the '-d <deviceId>' flag, or use '-d all' to act on all devices.

Pixel 6 (mobile)           • SOME_ANDROID_ID_BLABLA                  • android-arm64  • Android 13 (API 33)
techouse's iPhone (mobile) • SOME_IPHONE_ID_BLABLA  • ios            • iOS 16.3 20D47
macOS (desktop)            • macos                  • darwin-x64     • macOS 12.6 21G115 darwin-x64
Chrome (web)               • chrome                 • web-javascript • Google Chrome 123.456.789.123
```

**Comment by leoafarias** [comment]
@techouse This issue has been fixed on 3.0
